docs/scos/dev/migration-program/migration-to-paas/paas-migration-documents/transfer-environment-specific-application-settings-to-spryker-cloud-env-variables.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Transfer environment specific application settings to Spryker Cloud (ENV variables)
+description: This document describes how to transfer environment specific application settings to Spryker Cloud.
+template: howto-guide-template
+---
+
+# Transfer environment specific application settings to Spryker Cloud (ENV variables)
+
+{% info_block infoBox %}
+
+## Resources for assessment Backend, DevOps
+
+{% endinfo_block %}
+
+## If the ENV variables approach is not used
+
+Environment variables must be used for all the sensitive values such as 3rd party integration tokens, encryption keys, etc.
+
+For example this kind of configuration:
+```php
+$config[ServiceApiConstants::BASE_URL] = 'https://api.base.url';
+$config[ServiceApiConstants::API_KEY] = 'api-key-value';
+$config[ServiceApiConstants::API_SECRET] = 'api-secret-value';
+```
+should be changed to this:
+```php
+$config[ServiceApiConstants::BASE_URL] = env('SERVICE_API_BASE_URL');
+$config[ServiceApiConstants::API_KEY] = env('SERVICE_API_KEY');
+$config[ServiceApiConstants::API_SECRET] = env('SERVICE_API_SECRET');
+```
+All the variables must be created on Spryker Cloud and all the values must be added there for all the environments
+such as production, staging, sandboxes, etc.
+
+It also might be useful to go through module configuration to be sure all the needed configurations are located in
+the `config/Shared` folder.
+
+## If the ENV variables approach is used
+
+All the environment variables need to be created on Spryker Cloud for all the environments such as production, staging, sandboxes, etc.
+
+### Action
```

Comment:
Please rework this section. I don't understand how it's different from the previous steps in this docs, who's supposed to do it and what "instruction" you are referring to.